dom/events/delegate/delegate-test.js

```diff
@@ -5,6 +5,7 @@ var domEvents = require('can-util/dom/events/');
 var domDispatch = require('can-util/dom/dispatch/');
 var domData = require('can-util/dom/data/');
 require('can-util/dom/events/delegate/');
+require('can-util/dom/events/delegate/derive-enter-leave');
```

Comment:
this code derives enter and leave events from other events. i'm open to suggestions